docs/07-AdvancedUsage.md

```diff
@@ -331,353 +474,7 @@ groups:
-and in there you can just specify options that you wish to override:
-
-```yaml
-modules:
-    config:
-        WebDriver:
-            browser: 'chrome'
-```
-
-Environment configuration files are merged into the main configuration before suite configuration is merged.
-
-You can easily switch between those configs by running tests with `--env` option. To run tests only for PhantomJS you need to pass `--env phantom` option:
-
-```bash
-$ php codecept.phar run acceptance --env phantom
-```
-
-To run tests in all 3 browsers, just list all the environments:
-
-```bash
-$ php codecept.phar run acceptance --env phantom --env chrome --env firefox
-```
-
-and tests will be executed 3 times, each time in a different browser.
-
-It's also possible to merge multiple environments into one configuration by using comma as a separator:
-
-```bash
-$ php codecept.phar run acceptance --env dev,phantom --env dev,chrome --env dev,firefox
-```
-
-Configuration is merged in the given order. This way you can easily create multiple combinations of your environment configurations.
-
-Depending on environment you may choose which tests are to be executed.
-For example, you might need some tests to be executed only in Firefox, and few tests only in Chrome.
-
-Desired environments can be specified with `@env` annotation for tests in Test and Cest formats:
-
-```php
-<?php
-class UserCest
-{
-    /**
-     * This test will be executed only in 'firefox' and 'phantom' environments
-     *
-     * @env firefox
-     * @env phantom
-     */
-    public function webkitOnlyTest(AcceptanceTester $I)
-    {
-        // I do something
-    }
-}
-?>
-```
-
-For Cept you should use `$scenario->env()`:
-
-```php
-<?php
-$scenario->env('firefox');
-$scenario->env('phantom');
-// or
-$scenario->env(array('phantom', 'firefox'));
-?>
-```
-
-If merged environments are used, then you can specify multiple required environments (order is ignored):
-
-```php
-<?php
-$scenario->env('firefox,dev');
-$scenario->env('dev,phantom');
-?>
-```
-
-This way you can easily control what tests will be executed for which environments.
-
 
 ## Conclusion
 
-Codeception is a framework which may look simple at first glance. But it allows you to build powerful tests with single API, refactor them, and write them faster using interactive console. Codeception tests can be easily organized in groups or Cest classes, locators can be stored in PageObjects and common steps can be combined in StepObjects.
-
-Probably it has too much features for one framework. But nevertheless Codeception follows the KISS principle: it's easy to start, easy to learn, and easy to extend.
+Codeception is a framework which may look simple at first glance. But it allows you to build powerful tests with single API, refactor them, and write them faster using interactive console. Codeception tests can be easily organized in groups or Cest classes.
```

Comment:
powerful tests with **a** single API

faster using **the** interactive console